### ISSUE TITLE
Bug: dst not allowed in RUN mount from buildkit

### ISSUE THREAD

**Issue by johnmbw** [hadolint/hadolint#434]
- [x] This is a bug report
- [ ] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior

The `dst` param is a valid param for a secret mount (to specify the destination name):

https://docs.docker.com/develop/develop-images/build_enhancements/#new-docker-build-secret-information

e.g.
```
RUN --mount=type=secret,id=m2settings,dst=/settings.xml mkdir -p ~/.m2/ && cp /settings.xml ~/.m2/settings.xml
```

Hadolint should be ok with this parameter.

### Actual behavior

Hadolint tosses an error about an unexpected arg:

```
unexpected "dst=/setting" expecting "gid=", "id=", "mode=", "required", "uid=", source=, or target=
```

### Steps to reproduce the behavior

```bash
docker run --rm -i hadolint/hadolint:v1.17.7 < Dockerfile
/dev/stdin:3:39 unexpected "dst=/setting" expecting "gid=", "id=", "mode=", "required", "uid=", source=, or target= 
```

**Dockerfile** (if relevant)

```Dockerfile
# syntax=docker/dockerfile:experimental
FROM ubuntu:20.04 as base
RUN --mount=type=secret,id=m2settings,dst=/settings.xml mkdir -p ~/.m2/ && cp /settings.xml ~/.m2/settings.xml
```

**Additional environment details (OS, stack version, etc.)**

Mac OS X 10.14.3

**Comment by johnmbw** [comment]
NB. it looks like `destination` works fine though

**Comment by lorenzo** [comment]
heh, I implemented `dest` and `destination`, but it should have been `dst`. Please use `destination=` as a workaround while this gets fixed.

**Comment by johnmbw** [comment]
Yeah using destination is fine for me - easy enough to change.  Just thought I'd raise a ticket in case other people run into the issue.  Thanks for looking at it quickly 😄 .

**Comment by lorenzo** [comment]
this was fixed in the most recent release
